Documentation/artifact-format-v3.md

```diff
@@ -270,19 +270,17 @@ As an opposite to the list of global `artifact_provides` being a part of
 `header-info` file, the `artifact_provides` section in the `type-info` file
 is a set of parameters specific for a given payload type.
 
-The list of currently supported parameters is as follows:
-
-* `rootfs_image_checksum` is the checksum of the image contained within the
-  Artifact
+The `artifact_provides` is a key-value store, where the value is either a list,
+or an array of strings.
 
 #### artifact_depends
 
 The `artifact_depends` section in the `type-info` file is a set of parameters
 specific for a given payload type. The list of currently supported
 parameters is as follows:
```

Comment:
Maybe lose the last sentence here.

Reply:
You don't want the value type constraints mentioned at all?

Reply:
No, I do. I mean literally just this sentence: "The list of currently supported parameters is as follows:"